autokeras/tasks/time_series_forecaster.py

```diff
@@ -1,7 +1,118 @@
+import pandas as pd
+
 from autokeras import auto_model
+from autokeras import hypermodels
+from autokeras import nodes as input_module
+from autokeras.tuners import greedy
+
+
+class SupervisedTimeseriesDataPipeline(auto_model.AutoModel):
+
+    def __init__(self,
+                 outputs,
+                 column_names=None,
+                 column_types=None,
+                 lookback=None,
+                 predict_from=1,
+                 predict_until=10,
+                 **kwargs):
+        inputs = input_module.TimeseriesInput(lookback=lookback,
+                                              column_names=column_names,
+                                              column_types=column_types)
+        if column_types:
+            for column_type in column_types.values():
+                if column_type not in ['categorical', 'numerical']:
+                    raise ValueError(
+                        'Column_types should be either "categorical" '
+                        'or "numerical", but got {name}'.format(name=column_type))
+        if column_names and column_types:
+            for column_name in column_types:
+                if column_name not in column_names:
+                    raise ValueError('Column_names and column_types are '
+                                     'mismatched. Cannot find column name '
+                                     '{name} in the data.'.format(name=column_name))
```

Comment:
Can we extract this part and the same part in StructuredData into a function?

Reply:
Done